habitat-lab/habitat/gui/gui_application.py

```diff
@@ -178,12 +179,17 @@ def draw_event(self):
             self._last_draw_event_time = curr_time
             if self._debug_counter >= 100:
                 elapsed = curr_time - self._debug_timer
-                sps = self._debug_counter / elapsed
-                print("sps: {:.1f}".format(sps))
+                self._debug_sps = self._debug_counter / elapsed
+                print("[debug] sps: {:.1f}".format(self._debug_sps))
```

Comment:
We could just get rid of this console spam since you're printing on-screen, yeah?